src/ui/App.tsx

```diff
@@ -251,11 +276,11 @@ function App() {
   };
 
   const handleBulkAction = (action: ReleaseType) => {
-    const newSelections = { ...selections };
+    const newReleaseSelections = { ...selections };
     selectedPackages.forEach((packageName) => {
-      newSelections[packageName] = action;
+      newReleaseSelections[packageName] = action;
     });
-    setSelections(newSelections);
+    setSelections(newReleaseSelections);
```

Comment:
Are these left over from the rename commit?

``` suggestion
    const newSelections = { ...selections };
    selectedPackages.forEach((packageName) => {
      newSelections[packageName] = action;
    });
    setSelections(newSelections);
```

Reply:
Yes indeed I missed this one